From 013a9e1befb52abeed5cc8b9b93216ad3e2f83ca Mon Sep 17 00:00:00 2001
From: Yurii Rashkovskii <me@yrashk.com>
Date: Mon, 19 Feb 2018 01:08:29 +0700
Subject: [PATCH] Problem: in sit-web, comment text blends with the header

They are both white and this doesn't look good.

Solution: insert necessary styling to make comment header gray
---
 sit-web/webapp/src/issue/comment-style.html | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/sit-web/webapp/src/issue/comment-style.html b/sit-web/webapp/src/issue/comment-style.html
index 0e2cf89..1c7332e 100644
--- a/sit-web/webapp/src/issue/comment-style.html
+++ b/sit-web/webapp/src/issue/comment-style.html
@@ -10,7 +10,6 @@
             }
             rendered-text {
                 padding: 5pt;
-                background-color: #f0f0f0;
             }
 
             merge-request {
@@ -20,6 +19,9 @@
                 width: 100%;
                 --paper-card-header-text: {
                     font-size: 10pt;
+                };
+                --paper-card-header: {
+                    background-color: #f0f0f0;
                 }
             }
         </style>
-- 
2.15.0

